packages/opentelemetry-node/docs/supported-technologies.md

```diff
@@ -22,6 +22,16 @@ Dev Notes on supported Node.js versions:
   used by gaxios@6.7.0, a transitive dep of `@opentelemetry/resource-detector-gcp`.
 -->
 
+## TypeScript versions
+
+To use this package `@elastic/opentelemetry-node` with a TypeScript project
+requires TypeScript 4.7.2 or later.
+
+<!--
+Dev Notes on supported TypeScript versions:
+- v4.7.2 is required for support for `"module": "node16"` in tsconfig.json (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-7.html#ecmascript-module-support-in-nodejs)
+-->
+
```

Comment:
@david-luna Does this section seem reasonable? Should we mention the `module: "node16"` tsconfig.json requirement?

Reply:
commit 3685b63